addons/main/script_macros.hpp

```diff
@@ -134,7 +134,9 @@
 #define MRAD_TO_DEG(d) ((d) / 17.45329252) // Conversion factor: 9 / (50 * PI)
 #define MOA_TO_RAD(d) ((d) * 0.00029088) // Conversion factor: PI / 10800
 
-#define AI_DISPERSION(newDispersion,oldDispersion,oldCoef) ((oldDispersion * oldCoef) / (newDispersion))
+#define AI_DISPERSION(newDispersion,oldDispersion,oldCoefY,oldCoefX)\
+aiDispersionCoefX = (((oldDispersion) * (oldCoefX)) / (newDispersion));\
+aiDispersionCoefY = (((oldDispersion) * (oldCoefY)) / (newDispersion))\
```

Comment:
You don't need the parentheses not around single vars, because this expression is closed.

Reply:
The last escaped newline is wrong. May break if the line below is deleted during a reformatting. In any case, you just ended a line without ; which is logged as warning in RPT.

Reply:
Noticed and fixed with last commit.